spec/new-productions.htm

```diff
@@ -9,6 +9,18 @@
   </emu-grammar>
 </emu-clause>
 
+<emu-clause id="public-field-definition-early-errors">
+  <h1>Static Semantics: Early Errors</h1>
+    PublicFieldDefinition :
+      PropertyName[?Yield] Initializer?
+  <emu-grammar>
+  </emu-grammar>
+  <emu-alg>
+    1. It is a SyntaxError if |Initializer| contains |NewTarget|.
```

Comment:
Summarising the offline discussion I had with @jeffmo.

0. Agreed, we don't need to finalise the spec text for advancement to stage 2, but we do need to explain what we're trying to specify.
0. The `Contains` operation is different from the natural usage of "contains" in that it does not cross function boundaries.
0. @jeffmo claims his intent is for `new.target` to be disallowed within the initialiser, regardless of function boundaries; I think this is not a great decision.
0. @jeffmo claims his intent is for `arguments` to be disallowed only up to function boundaries (`Contains` semantics); I think this is a good decision.
0. We can't specify the `arguments` restriction in terms of `Contains`. We'll need a new spec operation (`ContainsArguments`?) to do this.